### ISSUE TITLE
Concordance, mise en place d'un "glossaire" (liste de mot clé d'une traduction)

### ISSUE THREAD

**Issue by Abderrazak** [benel/TraduXio#49]
Une fonctionnalité demandé par le client est la mise en place de glossaire "raisonné".

Afin de ne pas éclipser la fonctionnalité de recherche de concordance (un glossaire fixe une traduction pour un mot dans un texte, la recherche de concordance n'a alors plus lieu d'être), une solution proposé est la possibilité au moment de la traduction de définir par le traducteur une liste de "mot clé" du texte qui apparaitrait au moment de la recherche de concordance comme choix "prédéfini" pour l'utilisateur.

**Comment by tisbardo** [comment]
# Scénario
Pré-requis : Une oeuvre doit être présente dans la base, on doit être identifié en tant qu'utilisateur, et on doit avoir choisi une oeuvre.

## Ajouter une traduction dans le glossaire
Dans la zone de texte de l'oeuvre originale ou d'une traduction, 
* Je sélectionne un mot ou une phrase
* J'effectue un clic droit sur la sélection
* Je sélectionne "Ajouter une traduction dans le glossaire"
* Un bandeau s'affiche en haut de la page.
* Je sélectionne la langue de destination et la traduction.
* Je clique sur "Créer"

## Edition d'une traduction du glossaire
* Je fais un clic droit sur un texte surligné indiquant une entrée du glossaire.
* Une liste des traductions s'affiche
* Je clique sur le "e" correspondant à la traduction que je souhaites modifier.
* Je modifie la traduction dans le bandeau de la même façon qu'à l'ajout d'une traduction.

## Suppression
* Je fais un clic droit sur un texte surligné indiquant une entrée du glossaire.
* Une liste des traductions s'affiche
* Je clique sur le "X" correspondant à la traduction que je souhaites supprimer.

**Comment by benel** [comment]
Non, ce qui était prévu, c'était d'enregistrer une recherche dans la concordance.
